tbd/securityheaders/use.go

```diff
@@ -0,0 +1,16 @@
+package securityheaders
+
+import "net/http"
+
+func Use(h http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("Content-Security-Policy", "default-src 'self'")
+		w.Header().Add("Referrer-Policy", "same-origin")
+		w.Header().Add("Strict-Transport-Security", "max-age=31536000; includeSubDomains; preload")
+		w.Header().Add("X-Content-Type-Options", "nosniff")
+		w.Header().Add("X-Frame-Options", "SAMEORIGIN")
+		w.Header().Add("X-XSS-Protection", "1; mode=block")
```

Comment:
Worth adding the no-cache headers here as well, since we're going to be reusing those too.